<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceEdit</class>
 <widget class="QDialog" name="SurfaceEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="surfaceEditSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="buttonSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Knot vector type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="knotVectComboBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QFormLayout" name="formLayout_13"/>
            </item>
            <item>
             <widget class="QLabel" name="widthLabel">
              <property name="text">
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_15">
              <item row="0" column="0">
               <widget class="QLabel" name="orderLabel">
                <property name="text">
                 <string>Order:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="orderLineEdit">
                <property name="toolTip">
                 <string>Enter order</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QFormLayout" name="formLayout_10">
              <item row="0" column="0">
               <widget class="QLabel" name="xLabel_2">
                <property name="text">
                 <string>x:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="xSizeLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_11">
              <item row="0" column="0">
               <widget class="QLabel" name="yLabel_3">
                <property name="text">
                 <string>y:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="ySizeLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_12">
              <item row="0" column="0">
               <widget class="QLabel" name="zLabel_2">
                <property name="text">
                 <string>z:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="zSizeLineEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QLabel" name="colorLabel_2">
                <property name="text">
                 <string>Color</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <layout class="QFormLayout" name="formLayout_19">
                  <item row="0" column="0">
                   <widget class="QLabel" name="rLabel_3">
                    <property name="text">
                     <string>r:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="rLineEdit_2">
                    <property name="toolTip">
                     <string>Enter red value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout_20">
                  <item row="0" column="0">
                   <widget class="QLabel" name="gLabel_2">
                    <property name="text">
                     <string>g:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="gLineEdit_2">
                    <property name="toolTip">
                     <string>Enter green value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout_21">
                  <item row="0" column="0">
                   <widget class="QLabel" name="bLabel_2">
                    <property name="text">
                     <string>b:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="bLineEdit_2">
                    <property name="toolTip">
                     <string>Enter blue value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QLabel" name="positionLabel_2">
                <property name="text">
                 <string>Position</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <layout class="QFormLayout" name="formLayout_22">
                  <item row="0" column="0">
                   <widget class="QLabel" name="xLabel_4">
                    <property name="text">
                     <string>x:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="xLineEdit_2">
                    <property name="toolTip">
                     <string>Enter x value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout_23">
                  <item row="0" column="0">
                   <widget class="QLabel" name="yLabel_5">
                    <property name="text">
                     <string>y:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="yLineEdit_2">
                    <property name="toolTip">
                     <string>Enter y value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout_24">
                  <item row="0" column="0">
                   <widget class="QLabel" name="zLabel_4">
                    <property name="text">
                     <string>z:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="zLineEdit_2">
                    <property name="toolTip">
                     <string>Enter z value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <item>
               <widget class="QLabel" name="rotationLabel_2">
                <property name="text">
                 <string>Rotation</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <layout class="QFormLayout" name="formLayout_25">
                  <item row="0" column="0">
                   <widget class="QLabel" name="rLabel_4">
                    <property name="text">
                     <string>R:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="rollLineEdit_2">
                    <property name="toolTip">
                     <string>Enter roll value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout_26">
                  <item row="0" column="0">
                   <widget class="QLabel" name="pLabel_2">
                    <property name="text">
                     <string>P:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="pitchLineEdit_2">
                    <property name="toolTip">
                     <string>Enter pitch value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout_27">
                  <item row="0" column="0">
                   <widget class="QLabel" name="yLabel_6">
                    <property name="text">
                     <string>Y:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="yawLineEdit_2">
                    <property name="toolTip">
                     <string>Enter yaw value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="okButton">
            <property name="text">
             <string>Ok</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="listWidget"/>
     </widget>
     <widget class="QWidget" name="editSceneContainer" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
